src/main/resources/io/jenkins/plugins/sample/global_configuration/OnboardingPluginGlobalConfiguration/config.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:section title="Onboarding Plugin">
+        <f:entry title="${%Name}" field="name">
+            <f:textbox checkMethod="post" />
+        </f:entry>
+        <f:entry title="${%Description}" field="defaultSuffix">
+            <f:textarea checkMethod="post" />
```

Comment:
no needed as you are not adding the form validation here
```suggestion
            <f:textarea />
```

Reply:
Removed it